.github/workflows/ci.yml

```diff
@@ -3,11 +3,13 @@ name: CI
 on:
   push:
     branches:
-      - develop
```

Comment:
here

Reply:
Shouldn't we trigger the ci workflow when pushing to develop branch?